### ISSUE TITLE
collada importer should replace . or / with another for node name

### ISSUE THREAD

**Issue by volzhs** [godotengine/godot#9477]
**Operating system or device - Godot version:**
All

**Issue description:**

Blender often uses `.` character for a name.
But Godot does not allow `.` or `/` for node name.
~~~So better collada should replace it with something another.~~~
collada importer should replace . or / with another for node name

this might be related to #5808.

**Comment by akien-mga** [comment]
This should likely go to https://github.com/godotengine/collada-exporter/ ;)

**Comment by volzhs** [comment]
@akien-mga oh. should I close this one?

**Comment by akien-mga** [comment]
Unless something should be changed in the engine code, it's better to close this one and open a new one on the other repo yes.

**Comment by akien-mga** [comment]
Moved to https://github.com/godotengine/collada-exporter/issues/50

**Comment by bojidar-bg** [comment]
On the other hand, shouldn't we change the godot's collada importer instead, since collada seems to support names containing `.` and `/`?

**Comment by volzhs** [comment]
I thought that too just a few minutes ago. which one would be better?

**Comment by akien-mga** [comment]
Yes you're right, we should fix the importer, not the exporter.

**Comment by Calinou** [comment]
This appears to have been fixed in both 3.2.1 and the `master` branch.

Given the following structure in a Collada file:

![image](https://user-images.githubusercontent.com/180032/84594926-84b30780-ae55-11ea-8065-b4131acbb569.png)

Godot imports this:

![image](https://user-images.githubusercontent.com/180032/84594919-8086ea00-ae55-11ea-9ba0-4cb18ecfb2f7.png)

`Cube.....` is automatically renamed to `Cube2` because it would conflict with `Cube` after stripping disallowed characters.

Blender and Collada files: [collada_dots_slashes.zip](https://github.com/godotengine/godot/files/4776479/collada_dots_slashes.zip)
